.gitignore

```diff
@@ -14,6 +14,7 @@ hack/tools/bin
 test/e2e/data/infrastructure-docker/main/cluster-template*.yaml
 test/e2e/data/infrastructure-docker/v0.3/cluster-template*.yaml
 test/e2e/data/infrastructure-docker/v0.4/cluster-template*.yaml
+test/e2e/data/infrastructure-docker/v1.0/cluster-template*.yaml
```

Comment:
would it be best to make this more general going forward so we didn't have to manually maintain this with every release? e.g. `test/e2e/data/infrastructure-docker/**/cluster-template*.yaml`

Ref:

- https://git-scm.com/docs/gitignore

Reply:
Yup, thx!